### DISCUSSION TITLE
Error when running the `sync-path` script

### DISCUSSION THREAD

**Discussion by jbranchaud** [johnlindquist/kit#1269] [Errors]
![CleanShot 2023-06-01 at 10 41 55@2x](https://github.com/johnlindquist/kit/assets/694063/b55e6c68-fdab-49ee-9844-7be238e4ca1f)

One of my custom scripts stopped working recently. While investigating why, I came across the `sync-path` command in the _Kit Actions_ menu. However, when I ran it, I got this error.

`SyntaxError: Unexpected token '||='`

Though I don't think it is related, in case the extra context is helpful, this is the error I'm seeing when I try running my custom script.

<img width="1025" alt="CleanShot 2023-06-03 at 15 14 46@2x" src="https://github.com/johnlindquist/kit/assets/694063/763d74ea-a01e-4f67-a68a-fa79cc424a43">

**Comment by johnlindquist** [comment]
@jbranchaud Screenshare Monday?

getTabs/etc may need accessibility permissions to be able to run applescript against other apps...

I'll dig into the sync-path
